config/initializers/constants.rb

```diff
@@ -8,6 +8,7 @@
 STOCK_APP = "stock".freeze
 BROWSE_APP = "browse".freeze
 STOCKIT_APP = "stockit".freeze
+TOTAL_REQUESTS_STATES = ["submitted", "awaiting_dispatch", "closed", "cancelled"].freeze
```

Comment:
This should be in `app/models/orders.rb` as it is specific to orders.
Only general constants that are used elsewhere in the app should be here.

Reply:
Moved it to order_spec file.